lib/helpers.js

```diff
@@ -39,19 +39,11 @@ var helpers = module.exports = {
     // arrays into objects in the output.
     // We now check if an array is only in the child or parent, return that.
     if(_.isArray(b) && !a){
-      if(_.every(b, _.isObject)){
-        return _.merge(a,b);
-      } else {
-        return b;
-      }
+      return b;
     }
 
     if(_.isArray(a) && !b){
-      if(_.every(a, _.isObject)){
-        return _.merge(a,b);
-      } else {
-        return a;
-      }
+      return a;
```

Comment:
@rafaelkelly Can you please check if this should be changed? also check the associated test for more context (from commit 6eeb181a64ae2c4928f18ddacef5cbe2fb39df3a)